frontend/app/styles/secret-input.scss

```diff
@@ -1,15 +1,13 @@
 //  font from: https://github.com/noppa/text-security
 
 @font-face {
-  font-family: "text-security-disc";
-  src: url("/text-security-disc-compat.eot");
-  src: url("/text-security-disc-compat.eot?#iefix") format("embedded-opentype"),
-    url("/text-security-disc.woff2") format("woff2"),
-    url("/text-security-disc-compat.ttf") format("truetype");
+  font-family: text-security-disc;
+  src: url("https://raw.githubusercontent.com/noppa/text-security/master/dist/text-security-disc.woff");
```

Comment:
don't load the font from an external source ... try to serve a copy from cryptopus assets